spec/models/constituency_spec.rb

```diff
@@ -282,71 +254,4 @@
       URL
     end
   end
-
-  describe "#refresh!" do
-    context "when Parliament has dissolved" do
-      let!(:constituency) do
-        FactoryBot.create(:constituency, :coventry_north_east)
-      end
-
-      before do
-        stub_api_request_for("CV21PH").to_return(api_response(:ok, "coventry_north_east"))
-      end
-
-      it "updates the existing constituency" do
-        expect {
-          constituency.refresh!
-        }.to change {
-          constituency.reload.mp_name
-        }.from("Colleen Fletcher MP").to(nil)
-      end
-    end
-
-    context "when there is no example postcode" do
-      let!(:constituency) do
-        FactoryBot.create(:constituency, :coventry_north_east)
-      end
-
-      before do
-        allow(constituency).to receive(:example_postcode).and_return(nil)
-        allow(constituency).to receive(:example_postcode?).and_return(false)
-      end
-
-      it "doesn't update the constituency" do
-        expect {
-          constituency.refresh!
-        }.not_to change {
-          constituency.reload.mp_name
-        }
-      end
-    end
```

Comment:
looks like you removed the refresh job and the tests, but not the `#refresh!` and `.refresh!` methods

Reply:
Hmm, on reflection the job should remain and the tests reworded to reflect change after a by-election. However these methods will be reworked further since it needs to address Assembly Members and not MPs.

Reply:
Specs restored and reworded to the general case

Reply:
:sparkles: